app/models/osbb.rb

```diff
@@ -16,6 +16,17 @@ class Osbb < ApplicationRecord
   validates :email,  length: { maximum: 255 },
                      format: { with: VALID_EMAIL_REGEX,
                                message: 'format is not valid' }
+
+  searchkick word_middle: %i[name phone email website]
```

Comment:
move elasticsearch logic to a separate concern module
to isolate it from other model logic